cirq-google/cirq_google/calibration/phased_fsim.py

```diff
@@ -577,18 +584,24 @@ def from_moment(cls, moment: Moment, options: FloquetPhasedFSimCalibrationOption
 
     def to_calibration_layer(self) -> CalibrationLayer:
         circuit = Circuit([self.gate.on(*pair) for pair in self.pairs])
+        args: Dict[str, Any] = {
+            'est_theta': self.options.characterize_theta,
+            'est_zeta': self.options.characterize_zeta,
+            'est_chi': self.options.characterize_chi,
+            'est_gamma': self.options.characterize_gamma,
+            'est_phi': self.options.characterize_phi,
+            # Experimental option that should always be set to True.
+            'readout_corrections': True,
+        }
+        if self.options.readout_error_tolerance is not None:
+            args['readout_error_tolerance'] = self.options.readout_error_tolerance
+            args['correlated_readout_error_tolerance'] = _correlated_from_readout_tolerance(
```

Comment:
why do we need both if one is a function of the other?

Reply:
We probably don't need two of those. I added this because I didn't know what's the best way to go at the time of deploying this option server-side. This is hidden from the public API, it doesn't matter much; we can remove it in a future releases.